onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -682,7 +696,7 @@ def quantize_weight(self, weight, qType):
 
         return q_weight_name, zp_name, scale_name
 
-    def quantize_weight_per_channel(self, weight_name, weight_qType, channel_axis):
+    def quantize_weight_per_channel(self, weight_name, weight_qType, channel_axis, reduce_range=False):
```

Comment:
>reduce_range=False [](http://example.com/codeflow?start=83&length=18)

as we synced, please change it True by default because all ops calling quantize_weight_per_channel support reduce_range

Reply:
Sure, Thanks!